CRM/CiviAwards/Form/AwardReview.php

```diff
@@ -63,11 +63,27 @@ class CRM_CiviAwards_Form_AwardReview extends CRM_Core_Form {
    * {@inheritDoc}
    */
   public function setDefaultValues() {
-    if (empty($this->defaultValues) && $this->activityId) {
-      $this->defaultValues = $this->getProfileFieldValues();
+    $hasDefaultValues = !empty($this->defaultValues);
 
-      return $this->defaultValues;
+    if ($hasDefaultValues) {
+      return;
+    }
+
+    $this->defaultValues = [];
+    $isAddAction = $this->_action & CRM_Core_Action::ADD;
+
+    if ($this->activityId) {
+      $this->defaultValues = array_merge(
```

Comment:
Do we really need a merge here. Why not just `$this->defaultValues = $this->getProfileFieldValues()` ?